### PR TITLE
work done to add in tracking of cloudfront distributions and distribu…

### PR DESCRIPTION
**Pull request by riot-jetaylor**
…tion origins. need to work on origin groups

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,9 +42,6 @@ $(AUDITORS): build_dir
 build_cli: generate
 	go build -o ./cinqctl cmd/*.go
 
-generate:
```
Comment (rnikoopour) [inline Makefile:45]: Why was this removed?

**Review by HTTP500 — APPROVED** [review]
